pages/signin.js

```diff
@@ -90,10 +90,11 @@ class SigninPage extends React.Component {
   }
 
   async initialize() {
-    if (this.props.token) {
+    const token = this.props.token || this.props.router.query.token;
```

Comment:
I'm curious about why is this needed now?

Reply:
This makes the login links work on Vercel (to be able to test the workspace on preview deploys), but perhaps something to be removed and looked at in another PR. I'll remove it.